src/gpl/src/replace.tcl

```diff
@@ -263,7 +265,24 @@ proc global_placement { args } {
       gpl::replace_incremental_place_cmd
     } else {
       gpl::replace_initial_place_cmd
-      gpl::replace_nesterov_place_cmd
+
+      if { ![info exists flags(-skip_nesterov_place)] } {
+        catch {gpl::replace_nesterov_place_cmd} error
+        set block [ord::get_db_block]
+
+        set diverge_codes [list "0304" "0305" "0306" "0307"]
```

Comment:
This is a reasonable solution but it has me worried that this can break if the error codes change in the future. @maliberty any suggestions?